### ISSUE TITLE
PGo cannot parse -1

### ISSUE THREAD

**Issue by mrordinaire** [UBC-NSS/pgo#19]
The code below fails at `-`

```
---------- MODULE small -------------
(*
--algorithm Test {
  variables i = 1;
  {
  a:  while (i # -1) {
  b:    i := i - 1;
    };
  }
}
*)
===============================================
```

**Comment by mrordinaire** [comment]
@fhackett, could you please test if your TLA+ expr parser can handle this?

**Comment by mrordinaire** [comment]
Fixed in #32

**Comment by fhackett** [comment]
superseded by #70
